packages/react-components/src/components/PromoBannerV2/PromoBannerV2.module.scss

```diff
@@ -0,0 +1,78 @@
+$base-class: 'promo-banner-v2';
+
+.main-wrapper {
+  container-type: inline-size;
+}
+
+.#{$base-class} {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  border-radius: var(--radius-3);
+  background-color: var(--surface-accent-emphasis-low-info);
+
+  &__content {
+    display: flex;
+    flex: 0 1 auto;
+    flex-flow: column;
+    justify-content: center;
+    padding: var(--spacing-5);
+    max-width: 720px;
+
+    &__cta {
+      display: flex;
+      gap: var(--spacing-2);
+      margin-top: var(--spacing-4);
+    }
+  }
+
+  &__additional-content {
+    display: flex;
+    flex: 1 1 auto;
+    align-items: flex-end;
+    justify-content: right;
+    padding-right: var(--spacing-2);
+    padding-left: var(--spacing-3);
+
+    img {
+      max-width: 100%;
+      height: auto;
+      object-fit: contain;
+    }
+  }
+
+  &__close {
+    margin-left: auto;
+    padding-top: var(--spacing-3);
+    padding-right: var(--spacing-4);
+
+    &__btn {
+      border: 0;
+      background: 0;
+      padding: 0;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+
+  @container (max-width: 559px) {
```

Comment:
I'd suggest putting this number into `constants.ts` and later reuse it here and in `.mdx` file